### ISSUE TITLE
Validation loss stays constant

### ISSUE THREAD

**Issue by amndzr** [lyuwenyu/RT-DETR#204]
I uncommented the following code that calculates the validation loss.
![grafik](https://github.com/lyuwenyu/RT-DETR/assets/148044783/10c4530a-8d28-45e5-a863-f14163c0d5a6)

However, when training the model the validation loss stays constant and doesn't move a bit. I trained the model on the COCO Dataset and on my Custom Dataset and in both the validation loss stays the same.

e. g. COCO2017
![grafik](https://github.com/lyuwenyu/RT-DETR/assets/148044783/e688a8f1-ed78-434e-99f0-b433211b0195)

e. g. Custom Dataset (I removed the scaling on the val loss)
![grafik](https://github.com/lyuwenyu/RT-DETR/assets/148044783/cded234b-3e82-4d43-9389-472085abdaa2)

Im using the newest commit. Additionally, I newly cloned the repository when training on the COCO2017 dataset.

Any help is appreciated.
